AppCenter/AppCenter/MSACService.h

```diff
@@ -12,23 +12,11 @@
 @protocol MSACService <NSObject>
```

Comment:
NS_SWIFT_NAME?

Reply:
Tbh I doubt that it is going to be used, but I'll add it